.github/workflows/ci.yaml

```diff
@@ -40,6 +40,12 @@ jobs:
       - name: Typing (mypy)
         run: poetry run pre-commit run mypy --all-files
 
+      - name: Generate schema (kpops schema)
+        run: poetry run pre-commit run gen-schema --all-files --show-diff-on-failure
+
+      - name: Generate docs (typer-cli)
+        run: poetry run pre-commit run gen-docs --all-files --show-diff-on-failure
```

Comment:
great! 🎉

Reply:
doesn't the docs part also need the docs dependencies installed?

Reply:
It did, but in order to run the pre-commit locally and more importantly for it to fail, you needed to have them installed and I moved the `typer-cli` dependency to the dev dependencies.